app/styles/gavant-ember-modals.scss

```diff
@@ -57,7 +57,7 @@ $modal-z-index: 999999;
 }
 
 .modal-footer {
-    padding: 10px 20px 20px 20px;
+    padding: $modal-dialog-footer-padding;
     border-top: 0;
 }
```

Comment:
@bakerac4 thoughts on just removing all the stylistic/presentational CSS rules from the addon (except for the mandatory styles concerning the positioning/layout and animations) and just letting the Boostrap 4 modal styles be the default?

I think in most cases the template we choose will include its own styles for modals, and the default bootstrap 4 styles should be sufficient in most cases where it doesn't. 

It looks like we took most of these styles from C4, which are based on Material design, and will compete against any styles the template/app wants to use.

Specifically, I think it would be ok to get rid of the following rule blocks:
```
.modal-content
.modal-header
.modal-body
.modal-footer
.modal-btn
```